test/functional/lua/secure_spec.lua

```diff
@@ -168,4 +168,84 @@ describe('vim.secure', function()
       eq(false, curbufmeths.get_option('modifiable'))
     end)
   end)
+
+  describe('trust()', function()
+    local xstate = 'Xstate'
+
+    setup(function()
+      helpers.mkdir_p(xstate .. pathsep .. (iswin and 'nvim-data' or 'nvim'))
+    end)
+
+    teardown(function()
+      helpers.rmdir(xstate)
+    end)
+
+    before_each(function()
+      helpers.write_file('test_file', 'test')
+    end)
+
+    after_each(function()
+      os.remove('test_file')
+    end)
+
+    it('sets file as trusted then denied', function()
+      local cwd = funcs.getcwd()
+      local hash = funcs.sha256(helpers.read_file('test_file'))
+
+      eq({true, nil}, exec_lua([[return {vim.secure.trust('test_file', 'allow')}]]))
```

Comment:
Not sure if this is the correct way to assert on return values from a lua func? I've been looking through the test for a similar use-case and this is the only way to achieve this I found.

Reply:
Yep this looks fine.